### PR TITLE
Some cleaning

### PR DESCRIPTION
**Pull request by SPalominos**
Now adds all the script contained in the resource folder instead of loading them one by one. (issue #13)

### CODE REVIEW

File: scripts/src/main/java/org/orbiswps/scripts/WpsScriptPlugin.java [hunk 1]
```diff
@@ -145,45 +145,23 @@ public void unsetWpsClient(WpsClient wpsClient) {
      */
     @Activate
     public void activate(){
-        System.out.println("test");
         listIdProcess = new ArrayList<>();
         //Check the WpsService
         if(wpsServer != null){
+            //Mark the string to translate which will be used in the tree path.
+            I18N.tr("OrbisGIS");
```
Comment (ebocher) [inline scripts/src/main/java/org/orbiswps/scripts/WpsScriptPlugin.java]: Why not generate the text on the fly from the tree folder ?

Comment (nicolas-f) [inline scripts/src/main/java/org/orbiswps/scripts/WpsScriptPlugin.java]: Use the method `I18n.marktr( )` in order to mark for translation without doing the translation.

@ebocher The translation of folder names are done on the fly in the tree folder. However the string have to be added to transifex, so at least one reference of the string have to be in the code.

Comment (SPalominos) [inline scripts/src/main/java/org/orbiswps/scripts/WpsScriptPlugin.java]: Thank you @nicolas-f, the fix is done.

**Comment by nicolas-f** [comment]
You have to update this line too:
https://github.com/orbisgis/orbiswps/blob/master/scripts/pom.xml#L57

Add `-kmarktr` in order to parse the right method

**Comment by SPalominos** [comment]
@nicolas-f, change done ;)

**Pull request merged**